tests/sample_files.rs

```diff
@@ -0,0 +1,48 @@
+use assert_cmd::assert::Assert;
+use assert_cmd::Command;
+
+use std::io::Write;
+use tempfile::{NamedTempFile, TempPath};
+
+fn create_temp_file(contents: &str) -> TempPath {
+    let mut file = NamedTempFile::new().unwrap();
```

Comment:
Please use structs and functions from [cli_common.rs](https://github.com/dotenv-linter/dotenv-linter/blob/master/tests/cli_common.rs).

Reply:
There are 2 reasons why I didn't use it:
 * `TestFile::new` uses the `writeln!` macro, and it appends a new line to content (https://doc.rust-lang.org/std/macro.writeln.html).
* The idea was to check the correctness (or incorrectness) of the files without providing the exact error messages (and `TestDir::test_command_fail_with_args` requires such messages).

Reply:
Can I make some changes in `cli_common`?

Also, I can leave only the tests that relate to checking the blank line at the end of the file (and rename the test), if you don't like this approach (arrays of the right and wrong examples - I like it because it allows easy to add more examples).

Reply:
Done